### PR TITLE
Describe difference between customer group and customer segment

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request:
- Describes difference between customer group and customer segment
- Corrects a typo in a word

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- yes

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/customers.html
https://docs.magento.com/user-guide/marketing/customer-segments.html

whatsnew
Updated the [Customer Segments](https://docs.magento.com/user-guide/marketing/customer-segments.html) topic to include comparison with customer segments based on where each are used.

### CODE REVIEW

File: src/marketing/customer-segments.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ redirect_from:
 
 Customer segments allow you to dynamically display content and promotions to specific customers, based on properties such as customer address, order history, shopping cart contents, and so on. You can optimize marketing initiatives based on targeted segments with shopping cart price rules. You can also generate reports and export the list of targeted customers. Because customer segment information is constantly refreshed, customers can become associated and disassociated from a segment as they shop in your store.
 
+To better understand the difference between customer groups and customer segments and where they are used, see:
```
Comment (leslietilling) [inline src/marketing/customer-segments.md:11]:
```suggestion
To better understand the difference between [customer groups]({% link customers/customer-groups.md %}) and customer segments, note where they are used:
```

File: src/marketing/customer-segments.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ redirect_from:
...
+
+||Customer Segment|Customer Grop|
```
Comment (leslietilling) [inline src/marketing/customer-segments.md:13]:
```suggestion
||Customer segment|Customer group|
```

File: src/marketing/customer-segments.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ redirect_from:
...
+|--- |--- |--- |
+|Catalog Price Rule||✔️|
+|Cart Price Rule|✔️|✔️|
+|Tier Price||✔️|
+|Related Product Rule|✔️||
+|Dynamic Block|✔️||
+|Reward Exchange Rates||✔️|
+|Category Permissions||✔️|
```
Comment (leslietilling) [inline src/marketing/customer-segments.md:21]: We use sentence case for table headings and items

```suggestion
|Catalog price rule||✔️|
|Cart price rule|✔️|✔️|
|Tier price||✔️|
|Related product rule|✔️||
|Dynamic block|✔️||
|Reward exchange rates||✔️|
|Category permissions||✔️|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Add link and minor edits for style

**Comment by ZhovnirMaria** [comment]
@leslietilling I have added some changes, please check. thanks ☺️

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**